app/main/helpers.js

```diff
@@ -25,7 +25,7 @@ export function openBrowserWindow (route, opts) {
   // note that __dirname changes based on whether we're in dev or prod;
   // in dev it's the actual dirname of the file, in prod it's the root
   // of the project (where main.js is built), so switch accordingly
-  let htmlPath = path.resolve(__dirname,  isDev ? '..' : 'app', 'renderer', 'index.html');
+  let htmlPath = path.resolve(__dirname,  'dist', 'index.html');
```

Comment:
Since parcel uses `index.html` as it's entry point, `index.html` now gets copied over to `dist/`, so we use that file instead.

Reply:
finally, i hated this hack

Reply:
looks like we can do away with the comment above, now?

Reply:
wait so `__dirname` is always the root of the project now, not the directory of the current file?

Reply:
`helpers.js` gets bundled into `main.js`, which runs in the root.

Reply:
It used to be that either the main thread would be run from `main/main.development` for `dev` and `main.js` from `prod`. Now I just have always build `main.js` (because it's fast, and the performance isn't an issue).